lib/stbcbckt.gi

```diff
@@ -2643,8 +2643,8 @@ local Pr, div, B, rbase, data, N;
     N := ShallowCopy( G );
   fi;
   # remove cached information
-  Unbind(G!.suborbits);
-  Unbind(E!.suborbits);
+  G!.suborbits:=[];
+  E!.suborbits:=[];
```

Comment:
Note that this is correct because of line 466: if `!.suborbits` is missing, then it is immediately set to an empty list. So we may just as well set it to an empty list right away, instead of unbinding (as becomes clear when one inspects all three places accessing it -- all three are visible in the diff of this PR.